mmv1/products/networkservices/Gateway.yaml

```diff
@@ -120,6 +162,7 @@ properties:
   - !ruby/object:Api::Type::String
     name: 'scope'
     required: true
+    immutable: true
```

Comment:
what's the current behavior if users try to update the field? Will the API return errors in all cases? Or will it silently ignore the request in some cases? I understand that the field might need to set as immutable in the first place, as shown in the field description, but we should be extra cautious about adding `forcenew` to a field to ensure backward-compatibility, as now it will delete and recreate the resource.

Reply:
This field supposed not to be updatable from the first place. The apis throws an error when trying to update only the scope. And by adding immutable:true it forces to recreate the resource.
I managed to test this scenario and worked as described.